src/main/java/com/ericsson/ei/EndpointSecurity.java

```diff
@@ -84,7 +94,10 @@ protected void configure(HttpSecurity http) throws Exception {
 
     @Override
     public void configure(AuthenticationManagerBuilder auth) throws Exception {
-        if(ldapEnabled) {
+        if (ldapEnabled && !ldapServerList.isEmpty()) {
```

Comment:
1. if ldapServerList is a string but not a valid json then the code here will be run. We need to test what happens then.

2. I think that ldapServerList should be validated and if not valid then create a server with the old properties and send it to addLDAPServersFromList

Reply:
Since the request was from Eiffel architects it should be okay to change the ldap properties to the new ldapserverlist properties and stop supporting old ldap properties format.
It is really bad to have two formats of properties,, we already have bad example of that i RemRem-Publish when something change and its impact configurations, developer forgot to test make it work for both format,, so now old properties format is to be removed RemRem-publihs due to the issues. So better to use one format directly and inform in release notes about these properties has changed.